### DISCUSSION TITLE
Icons and Bind Text Have Gone Bad in New Version

### DISCUSSION THREAD

**Discussion by BohdanPenchukov** [SFX-WoW/Masque#201] [Issues] [answered]
![Screenshot_1411](https://user-images.githubusercontent.com/23499600/64655594-068c8500-d42d-11e9-9554-97849889c5fa.png)
How it was before.
![Screenshot_21123](https://user-images.githubusercontent.com/23499600/64655634-1dcb7280-d42d-11e9-8812-fae7dd395c2a.png)
Now. Addon Dominos. Tried to go back to the last version, but there just standard. How to fix?

**Comment by StormFX** [comment, marked as answer]
> - This update contains extensive changes that will require add-on, **skin** and localization updates. **Please make sure that your add-ons and skins have been updated before reporting issues.**

https://github.com/StormFX/Masque/blob/master/CHANGELOG.md

**Comment by BohdanPenchukov** [comment]
So there's no way to fix it?

**Comment by StormFX** [comment]
"Gone bad" doesn't tell me anything. Neither does your images. **Tell me** the problem(s), don't expect me to figure it out. My previous comment points out that if you're using a skin that hasn't been updated to the latest version of _Masque_, **the skin will need to be updated**.

**Comment by BohdanPenchukov** [comment]
I think I understand. I'm using a Shadow skin that hasn't been updated since Draenor. The problem is that the text above the icon has become a layer below. Look image

**Comment by BohdanPenchukov** [comment]
But I think the skin will no longer be updated ever. Sorry to bother you

**Comment by StormFX** [comment]
If I get a moment, I'll look at this skin and see if I can do a quick fix. Do you have a link to the skin or a local copy?

**Comment by BohdanPenchukov** [comment]
https://www.curseforge.com/wow/addons/masque-shadow here it is my dude

**Comment by StormFX** [comment]
The only texture I see that appears above the hotkey text is the Gloss. Disable it.

**Comment by BohdanPenchukov** [comment]
ye dark gloss. Shadow, but thhen i will have just default icons if disable it) 
used to this skin :(

**Comment by StormFX** [comment]
Go to line 531 of Settings.lua in the Masque_Shadow folder and add the following line:

```lua
		DrawLayer = "OVERLAY",
```

**Comment by BohdanPenchukov** [comment]
![Screenshot_212](https://user-images.githubusercontent.com/23499600/64661209-16fc2a00-d444-11e9-9e19-eac78d9d0db7.png)

right?

**Comment by StormFX** [comment]
Yep.

**Comment by BohdanPenchukov** [comment]
Not changed. Settings file saved. Idk

**Comment by StormFX** [comment]
Before:
![Untitled-2](https://user-images.githubusercontent.com/1234928/64661759-93bee200-d40b-11e9-9ea6-bf2ccec8ea3b.jpg)
After:
![Untitled-3](https://user-images.githubusercontent.com/1234928/64661767-9ae5f000-d40b-11e9-9c40-a8d6b068fb03.jpg)

**Comment by StormFX** [comment]
Assuming you're using "Shadow 5". Otherwise, you'll need to add that line to the HotKey section of whatever skin you're using.

**Comment by BohdanPenchukov** [comment]
I see now. Using Shadow 2. Added to section 2. But dont works. May be im stupid xD 
![Screenshot_212](https://user-images.githubusercontent.com/23499600/64662657-77419a80-d449-11e9-961f-2cd164a68360.png)

**Comment by StormFX** [comment]
I switch to Shadow 2 and it's working fine after adding that line. Make sure you're reloading the UI. :p 
`/rl`

**Comment by StormFX** [comment]
Closing this as skin-related.
